AUTHORS.rst

```diff
@@ -45,4 +46,3 @@ Alphabetical list of contributors
 please send an email to the coordinators, or
 `open a pull request for this page <https://github.com/astropy/ccdproc/edit/master/AUTHORS.rst>`_
 in the `ccdproc repository <https://github.com/astropy/ccdproc>`_)
-
```

Comment:
That must be Atom being weird. I have faced one too many issues with empty newlines.